checkov/arm/checks/resource/AppServicePythonVersion.py

```diff
@@ -0,0 +1,27 @@
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class AppServicePythonVersion(BaseResourceValueCheck):
+
+    def __init__(self) -> None:
+        name = "Ensure that 'Python version' is the latest, if used to run the web app"
+        id = "CKV_AZURE_82"
+        supported_resources = ("Microsoft.Web/sites",)
+        categories = (CheckCategories.GENERAL_SECURITY,)
+        super().__init__(
+            name=name,
+            id=id,
+            categories=categories,
+            supported_resources=supported_resources,
+            missing_block_result=CheckResult.UNKNOWN)
+
+    def get_inspected_key(self) -> str:
+        return "properties/siteConfig/pythonVersion"
+
+    def get_expected_value(self) -> str:
+        return "3.4"
```

Comment:
@tsmithv11 Should we use 3.12 here?

Reply:
We should accept 3.9-3.12 here (all not EOL or close to EOL). The TF policy is on a deprecated resource that can only go up to 3.4, but this one appears to be able to go up to 3.12.

Reply:
So, @tsmithv11 correct me if I'm wrong, but it can be a Py check but a Yaml check, right?

Reply:
@ChanochShayner this is a single attribute inside a single resource, so a Python check will probably be better so we can have auto-fixes for it.